### PR TITLE
RPC 0.2.0 - Dev's Guide

### PR DESCRIPTION
**Pull request by Starzu**
@grzesiul take a look, please.

### CODE REVIEW

File: frontend/src/main/scala/io/udash/guide/views/bootstrapping/BootstrappingRpcView.scala [hunk 1]
```diff
@@ -20,46 +20,53 @@ class BootstrappingRpcView extends View {
...
     p(
       "That is all you have to do in the ", i("shared"), " module. Implementation of those interfaces will be covered in ",
       a(href := BootstrappingBackendState.url)("backend"), " and ", a(href := BootstrappingFrontendState.url)("frontend"),
       " bootstrapping chapters."
     ),
-    h3("RPC vs ClientRPC"),
+    h3("RPC vs Client RPC"),
     p(
-      i("RPC"), " is a marker trait for all RPC interfaces. A RPC interface is a trait or class " +
+      i("com.avsystem.commons.rpc.RPC"), " is an annotation marking all RPC interfaces. A RPC interface is a trait or class " +
```
Comment (grzesiul) [inline frontend/src/main/scala/io/udash/guide/views/bootstrapping/BootstrappingRpcView.scala]: "An RPC", I think we choose that phonetically

File: frontend/src/main/scala/io/udash/guide/views/Versions.scala [hunk 2]
```diff
@@ -1,12 +1,12 @@
 package io.udash.guide.views
 
 object Versions {
-  val udashCoreVersion = "0.1.1"
-  val udashRpcVersion = "0.1.0"
-  val udashJQueryVersion = "0.2.0"
+  val udashCoreVersion = "0.2.0-rc.1"
+  val udashRpcVersion = "0.2.0-rc."
```
Comment (grzesiul) [inline frontend/src/main/scala/io/udash/guide/views/Versions.scala]: shouldn't there be 1 or 2 after the dot?

**Comment by grzesiul** [comment]
While I'm planning to prepare PR with extended examples what allows GenCodec, I want you to review those minor issues.

**Comment by Starzu** [comment]
@grzesiul - done :)

**Comment by grzesiul** [comment]
lgtm

**Pull request merged**